packages/dev/core/src/Cameras/camera.ts

```diff
@@ -117,6 +118,18 @@ export class Camera extends Node {
      */
     public inputs: CameraInputsManager<Camera>;
 
+    /** @internal */
+    protected _lastUpdatedTime: number;
+
+    /** @internal */
+    protected _currentDeltaTime;
+
+    /**
+     * Delta time, in ms, for a frame run at 60 fps.
+     * @internal
+     * */
+    protected _standardDeltaTime = 1000 / 60;

```

Comment:
maybe even a part of the engine's Constants (ReferenceTimeStep or something similar?) as this is used throughout the framework as a reference.
Unifying this could be a task for a future PR, but in many sections we are using 1 / 60 (timestep in seconds) and in some we use 1000 / 60 (in ms)

Reply:
I'll create a constant and reference it in the places that I see we're doing the 1000 / 60 math.  As far as unifying things, I'll leave that as a future task.